articles/communication-services/quickstarts/email/enable-user-engagement-tracking.md

```diff
@@ -37,6 +37,9 @@ In this quick start, you'll learn about how to enable user engagement tracking f
 **Your email domain is now ready to send emails with user engagement tracking. Please be aware that user engagement tracking is applicable to HTML content and will not function if you submit the payload in plaintext.**
 
 You can now subscribe to Email User Engagement operational logs - provides information related to 'open' and 'click' user engagement metrics for messages sent from the Email service.
+> [!NOTE]
+> User Engagement Tracking cannot be enabled for Azure Managed Domains and can only be enabled for Custom Domains which have a sending limit of over 30 requests per minute and 100 requests per hour.
```

Comment:
`User Engagement Tracking cannot be enabled for Azure Managed Domains or Custom Domains with default sending limits. See [here](https://learn.microsoft.com/en-us/azure/communication-services/concepts/service-limits#rate-limits) for more info on limits.`